pyproject.toml

```diff
@@ -79,3 +79,9 @@ lint.unfixable = [
 "test/**" = [
     "B018",  # useless-expression
 ]
+
+[tool.codespell]
+skip = 'test/fixtures/reflog_*'
+ignore-words-list="gud,doesnt"
+#count = true
+quiet-level = 3
```

Comment:
sure, could you pls check #1920 and #1921

Reply:
Thanks! I've reviewed both those PRs.

#1920 is failing because, even where it intuitively feels like there would be no problem, even small stylistic changes to test data can cause tests to fail. This is true in general, but even more so for Git-related test data where the tests may be asserting something about the *hash* of the data or something directly or indirectly derived from it (though that is not the only mechanism through unexpected effects on tests can occur).

As detailed in my review there, such changes shouldn't be made lightly, sometimes the problems are *not* readily detectable by an immediate test failure, and most files in `test/fixtures` should not be style-checked or edited in this way.

I mention that here because this also applies to some of the changes here in this PR, as noted in https://github.com/gitpython-developers/GitPython/pull/1888#discussion_r1615317787.